linelist_investigations/report_sources/age_sex_2019-10-18.Rmd

```diff
@@ -165,13 +165,16 @@ We retain only entries:
 * corresponding to confirmed and probable cases
 * whose sex is known
 * whose age class is known
+* from 2019-01-01 to present
 
 ```{r filter}
 
 x <- x %>%
   filter(epicasedef %in% c("confirmed", "probable"),
          sex %in% c("male", "female"),
-         age_class != "unknown")
+         age_class != "unknown",
+         month_report >= as.Date("2019-01-01"))

```

Comment:
This works but safer to use the date of reporting (I think `date_report`) as `month_report` is derived from it.

Reply:
updated code accordingly